tensorflow/lite/toco/graph_transformations/resolve_constant_unary.cc

```diff
@@ -184,6 +184,7 @@ ::tensorflow::Status ResolveConstantUnaryOperator::Run(Model* model,
   CHECK(input_array.buffer);
   std::vector<DataType<ArrayDataType::kFloat>> const* input_float_data;
```

Comment:
To stay consistent with other areas (identify_l2_normalization.cc), you can initialize this to nullptr instead of setting it to nullptr in the branch.